azure-pipelines.yml

```diff
@@ -92,12 +92,6 @@ jobs:
         condition: succeeded()
         workingDirectory: '$(AppHostingSdkProjectDirectory)'
 
-      - task: Yarn@2
-        displayName: 'Run tests for electron webview'
-        inputs:
-          Arguments: 'run xvfb-maybe mocha'
-          ProjectDirectory: '$(AppHostingSdkProjectDirectory)'
-
```

Comment:
Why are we removing this?

Reply:
The answer was in the description "Keep the native tests in only one job of E2E test to remove redundancy." Resolving.